Mathlib/RingTheory/LocalRing/ResidueField/Instances.lean

```diff
@@ -0,0 +1,45 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.RingTheory.LocalRing.ResidueField.Ideal
+import Mathlib.FieldTheory.Separable
+
+/-! # Instances on residue fields -/
+
+variable {R A B : Type*} [CommRing R] [CommRing A] [CommRing B] [Algebra R A] [Algebra A B]
+    [Algebra R B] [IsScalarTower R A B]
+
+variable (p : Ideal A) [p.IsMaximal] (q : Ideal B) [q.IsMaximal] [q.LiesOver p]
+
+attribute [local instance] Ideal.Quotient.field
+
+instance [Algebra.IsSeparable (A ⧸ p) (B ⧸ q)] :
+    Algebra.IsSeparable p.ResidueField q.ResidueField := by
+  refine Algebra.IsSeparable.of_equiv_equiv
+    (.ofBijective _ p.bijective_algebraMap_quotient_residueField)
+    (.ofBijective _ q.bijective_algebraMap_quotient_residueField) ?_
+  ext x
+  simp [RingHom.algebraMap_toAlgebra, Algebra.ofId_apply]
```

Comment:
Non terminal `simp'. I guess we want a lemma simplifying `(algebraMap (A ⧸ p) (B ⧸ q)) ((Ideal.Quotient.mk p) x)` to close the goal immediately. (Similarly below)